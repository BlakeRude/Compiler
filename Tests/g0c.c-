// C-S21
int cat(int x) return x+x;

main()
{
    output(cat(88));
}
