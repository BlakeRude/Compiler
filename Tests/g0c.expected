Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: g0c.tm
176
Bye.
